s30566-DanielMakowski/zajecia2/zadanie6.txt

```diff
@@ -0,0 +1,22 @@
+Teraz postaraj się zrobić to zadanie z haczykiem i podmianą fragmentu zmiennej (Ala ma ...)
+
+Rozwiązanie:
+
+#!/bin/bash
+
+A=$1
+B=$( echo "${A: -1}")
+
+if [ "$B" = "a" ]; then
+        A2=$(echo "$A" | sed 's/a$/ę/g' | sed 's/\([^ę]\)$/\1a/g')
+        echo "$A ma kota, a kot ma ${A2}"
```

Comment:
to echo jak i to poniżej w wariancie else nie różnią się od siebie niczym. Można zatem przenieść to poza blok if-fi.